src/handlers/handler_token.py

```diff
@@ -41,10 +42,31 @@ class HandlerToken:
     HandlerToken manages token provider URL and public keys for JWT verification.
     """
 
+    _REFRESH_INTERVAL = timedelta(minutes=30)
```

Comment:
Probably little less to be on a safer side would be better, 28 minutes for example.